src/main/java/seedu/address/logic/commands/ListingAddCommand.java

```diff
@@ -0,0 +1,107 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Listing;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.UniqueListingList;
+import seedu.address.model.person.exceptions.DuplicateListingException;
+
+/**
+ * Changes the remark of an existing person in the address book.
+ */
+public class ListingAddCommand extends Command {
+
+    public static final String COMMAND_WORD_PREFIX = "listing";
+    public static final String COMMAND_WORD_SUFFIX = "add";
+    public static final String COMMAND_WORD = COMMAND_WORD_PREFIX + " " + COMMAND_WORD_SUFFIX;
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Adds a property listing to a Person "
+            + "by the index number passed into INDEX parameter. "
+            + "Multiple types of each property can exist, but the address must be unique.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "t/PROPERTY_TYPE a/ADDRESS\n"
+            + "Example: " + COMMAND_WORD + " 1 "
+            + "t/hdb a/NUS avenue 123";
+
+    public static final String MESSAGE_ADDED_LISTING_SUCCESS = "Added property listing to Person: %1$s";
+    public static final String HELP_LISTING_ADD_COMMAND = "Property Listing Add Command\n"
+            + "- Format: listing add INDEX t/PROPERTY_TYPE a/address\n"
+            + "- Example: listing add 1 t/hdb a/123 NUS Street\n"
+            + "- The list of valid PROPERTY_TYPE are: HDB, CONDO, LANDED";
```

Comment:
Great that you covered the basic property types. I think we can use AI to generate even more property types we haven't thought of.

Reply:
heya i think we are prob gonna stick with these 2 first since it demonstrates the functionality